lib/opencensus/attributes.ex

```diff
@@ -0,0 +1,232 @@
+defmodule Opencensus.Attributes do
+  @moduledoc """
+  Types and functions for Opencensus-compatibile span attributes.
+
+  To be compatible with the OpenCensus protobuf protocol, an [attribute value][AttributeValue]
+  [MUST] be one of:
+
+  * `TruncatableString`
+  * `int64`
+  * `bool_value`
+  * `double_value`
+
+  Some destinations are even stricter, e.g. Datadog. TODO: cite.
+
+  Despite its type declarations, `:ocp.put_attribute/2` in 0.9.1 is even stricter:
+
+      iex> :ocp.with_child_span("name")
+      iex> :ocp.put_attribute("double", 1.0)
+      {:error, :invalid_attribute}
+
+  ... but none of the others restrict attribute values, including:
```

Comment:
Instead of doc'ing this here we should have these fixed in opencensus and remove this part of these docs.

Reply:
That'd be nice. How good are they with resolving such issues? I'd expect it to get backed up indefinitely due to the possibility of breaking existing code. Might get fixed in the new combined projects, though. How's the competition to claim slots in the new namespace going?

I'll delete the “... but none of the others” paragraph and bullet list, and pursue the matter upstream.

Reply:
For now just open an issue on the opencensus-erlang repo and I'll get it fixed.

Reply:
census-instrumentation/opencensus-erlang#156

Reply:
About to push edit. You're right: it's better without the complaining.